piinguin-server/0.1.0-rc1/docker-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/dumb-init /bin/sh
+set -e
+
+# Make sure we run the server as the snowplow user
+exec su-exec snowplow:snowplow /usr/bin/java \
+  $SP_JAVA_OPTS -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap \
+  -jar ${SNOWPLOW_BIN_PATH}/piinguin-server-assembly-${PIINGUIN_SERVER_VERSION}.jar "$@"
```

Comment:
we usually remove `assembly` from the jar name, e.g. https://github.com/snowplow/snowplow/blob/master/2-collectors/scala-stream-collector/project/BuildSettings.scala#L47 but otherwise it looks good yes